src/org/datafoodconsortium/connector/codegen/ruby/static/spec/parse_with_skos_concept_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'datafoodconsortium/connector/data_reading_helper'
+
+describe "parse with skos concept" do
+  let(:connector) { DataFoodConsortium::Connector::Connector.instance }
+
+  describe "productTypes" do
+    before { connector.loadProductTypes(parse_json_file("productTypes.json")) }
+
+    describe "topConcepts" do
+      it "returns a list of available topConcepts" do
+        product_types = connector.PRODUCT_TYPES.topConcepts
+
+        expected = [
+          :BAKERY, :DAIRY_PRODUCT, :DRINK, :FROZEN, :FRUIT, :INEDIBLE, :LOCAL_GROCERY_STORE,
+          :MEAT_PRODUCT, :VEGETABLE
+        ]
+        expect(product_types).to eq(expected)
+      end
+    end
+
+    it "parses the first level" do
+      drink_type = connector.PRODUCT_TYPES.DRINK
+
+      expect(drink_type.is_a?(DataFoodConsortium::Connector::SKOSConcept)).to be(true)
```

Comment:
Rspec has better matchers for this which have better output when the spec fails:
```suggestion
      expect(drink_type).to be_a DataFoodConsortium::Connector::SKOSConcept
```